.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
+on: [ push ]
```

Comment:
Why run this on every push?

Reply:
It's just one build  way，I test it on my repo so,  I only provide a way to build,  with makefile to build rather than release flow , everything can be changed, I have no idea with the release flow ，so I set it as draft